### DISCUSSION TITLE
Google Cloud + Firebase doesn't allow app registrations using the same package name and SHA-1

### DISCUSSION THREAD

**Discussion by samfweb** [CariusLars/ar_flutter_plugin#39] [Q&A] [answered]
Thanks for your work on this! I'm a complete Google Cloud and Flutter beginner.

I'm attempting to follow your guide [here](https://github.com/CariusLars/ar_flutter_plugin/blob/main/cloudAnchorSetup.md) with the example provided however once I register an OAuth2.0 project with the debug SHA-1 I can't use these later during the Firebase setup step without Google Cloud throwing an error. 
![image](https://user-images.githubusercontent.com/44712417/135741257-5e243062-6d47-4337-b80e-a0a8f255a9eb.png)

Should I be using a different SHA-1 or package name?

**Comment by CariusLars** [comment, marked as answer]
Hey @samfweb , the error sounds like you already have a set of OAAuth credentials with the same package name and SHA-1 signing key. As I understand you went through the first part of the guide ("Set up Google Cloud Anchor Service") without problems but in the second part ("Set up Firebase") the error appeared, right?
You can always try changing your app's package name (make sure `AndroidMainfest.xml`) matches, but I think the steps in the guide should work, the error sounds more like you're accidentally trying to create a second set of OAuth2.0 credentials, so I'm trying to better understand in which step it occurs

**Comment by samfweb** [comment]
Thanks for the help and apologies for very late response. Using a new package name worked!
